README.adoc

```diff
@@ -103,6 +105,56 @@ class MyComponent @Inject() (mailerClient: MailerClient) {
 }
 ```
 
+#### With compile-time injection
+
+```scala
+import play.api.libs.mailer._
+import javax.inject.Inject
+import java.io.File
+import org.apache.commons.mail.EmailAttachment
+
+class MyComponent(mailerClient: MailerClient) {
+
+  def sendEmail {
```

Comment:
I think we should avoid the duplication of the `sendEmail` method and provide a little more context on Runtime / Compile time DI.